app/models/classroom.rb

```diff
@@ -1,5 +1,5 @@
 class Classroom < ApplicationRecord
   belongs_to :year
   belongs_to :school
-  has_many :users
+  has_many :users, dependent: :nullify
```

Comment:
I thought through the alternative arguments to `dependent:` and I think this is the correct one

Reply:
I also suspect we may end up with a join model here.

Reply:
🤔 Yeah, I can see that